package.json

```diff
@@ -18,6 +18,7 @@
     "test_unit": "gulp test:scripts:unit:watch",
     "test_unit_no_watch": "gulp test:scripts:unit",
     "test_functional": "chimp tests/functional/chimp.js",
+    "test_census": "./node_modules/cypress/bin/cypress open",
```

Comment:
Can we point this at the spec file by default because at the moment it open a UI and requires another click to run the spec. We need to able to use this in CI potentially?

Can we also configure it so cypress runs in headless mode when `EQ_RUN_FUNCTIONAL_TESTS_HEADLESS` is set, same way the current functional test works?

Reply:
Yes, we can. Resolved.